nip05.ts

```diff
@@ -1,22 +1,21 @@
+import crossFetch from 'cross-fetch'
```

Comment:
Isn't this going to include this dependency for everybody regardless of the environment they are?

Reply:
yeah you're right. reverting this back to how it was bc feels out of scope here. will resurface in a separate PR later

Reply:
[done](https://github.com/nbd-wtf/nostr-tools/compare/4fd83691968a7dacbc179fd08a571b4865329493..829e24b5bc65725b76172d764316e146e4ebe572)

using `cross-fetch` in test instead of `node-fetch` to support running same test in browser and node. [relevant commit](https://github.com/nbd-wtf/nostr-tools/pull/111/commits/0d577c67fd414ef26ea546efe7984a2e9849b5cb)